app/Pods/Fragaria/SMLTextView.m

```diff
@@ -1267,6 +1267,15 @@ - (NSRange)rangeForUserCompletion
     {
         return NSMakeRange(NSNotFound, 0);
     }
+    
```

Comment:
so this is the bit that worries me, the next time we do a `pod install` this gets overwritten.

In https://github.com/CocoaPods/CocoaPods-app/pull/333 there are some (now merged into master) functions added that may be able to replicate some of this work without having to make changes at SMLTextView level, any chance you can take a quick look? I'd like to get this merged so we can start looking at it more holistically for https://github.com/CocoaPods/CocoaPods-app/issues/311 and building an AST. It's easier to work with these things in place first IMO 👍